### DISCUSSION TITLE
Navigation Details (Destination, ETC etc)

### DISCUSSION THREAD

**Discussion by deggle** [teslamate-org/teslamate#3018] [Ideas]
I believe that the current navigation destination and related information is now included in the Tesla API (it's shown in the app) - while not directly useful for the TeslaMate dashboards etc, it could be useful to have this information available via the MQTT output.

Would this be useful for anyone else, and if so could we make a feature suggestion?

**Comment by mjdyson** [comment]
I'd find this useful to build a notifications in HomeAssistant...
when leaving work (+/- radius on lat/long) and navigation set to home location and vehicle speed > 0, send notification to contact with ETA.
update if ETA changes by more than 5 minutes. etc.

**Reply by craxo** [reply]
Coming from Tesla Custom Integration I second this, as I miss the ETA when navigating.

**Comment by vincep5** [comment]
Yes, this idea was partially committed with the change in https://github.com/teslamate-org/teslamate/pull/3657
However the MQTT topics were only included for the route's Destination, Lat, and Long.
I am hopeful that the committer or someone can add in the Distance to arrival and Minutes to arrival

**Comment by vincep5** [comment]
Would be a nice to have if MQTT had these published
```
"active_route_miles_to_arrival": 6.485299,
"active_route_minutes_to_arrival": 23.466667,
```

**Reply by hansie-p88** [reply]
I'm also very much interested in having this information available in HA via MQTT.

**Comment by brianmay** [comment]
See pull request #3789.

**Comment by deggle** [comment]
I was just thinking about the nav data so googled "teslamate nav" to find this thread... which I posted in 2022!
I'd completely forgotten I'd asked about this!

And here it is, active and being progressed as of 4 days ago - amazing!
Thank you @brianmay!
